examples/onboard-azure-ms-sql-server/README.md

```diff
@@ -0,0 +1,48 @@
+# Onboard Azure SQL Server example
+This example includes additional prerequisites that will need to be completed to fully utilize the module. More details can be found in the [onboarding documentation](https://docs.imperva.com/bundle/onboarding-databases-to-sonar-reference-guide/page/Azure-SQL-Server-Onboarding-Steps_48367377.html).
+
+This example creates both 'azurerm' and 'dsfhub' resources. More information regarding authentication to each can be found in the relevant provider documentation:
+- [azurerm](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs)
+- [dsfhub](https://registry.terraform.io/providers/imperva/dsfhub/latest/docs)
+
+## Prerequisites
+Both of the following prerequisites are handled within the ``onboard-azure-eventhub`` module.
+
+### Azure Event Hub Namespace and Event Hub
+SQL Server audit logs are sent to an Azure Event Hub and are retrieved by DSF. The Event Hubs are created within an Event Hub Namespace, which can contain one or more Event Hubs. Audit logs of multiple SQL Server instances can be sent to a single Event Hub. This module expects an Event Hub to have been created in advance, and with a corresponding AZURE EVENTHUB asset onboarded for it.
```

Comment:
I think I missed this in the other PR, but we could add a blurb about about the `onboard-azure-eventhub` module after the "This module expects an Event Hub to have been created in advance", like "In this example, we use the `onboard-azure-eventhub` module..."

Reply:
Rethinking it a bit, what do you think about moving the general idea of "This module expects an Event Hub to have been created in advance" to under Prerequisites since it applies to both the Event Hub and Storage Account Container? It could be something like:

"This module expects an Event Hub and a Storage Account Container to have been created in advance, in addition to a corresponding existing AZURE EVENTHUB asset on DSF. Both of these prerequisites and all related resources are handled in the ``onboard-azure-eventhub`` module."

And then the two sections would just remain descriptions of the Azure resources and how they're used by DSF

Reply:
Sure, sounds good to me.